### PR TITLE
fix(app): ignore lpc commands when reconciling run and analysis for run log

### PR DESCRIPTION
**Pull request by b-cooper**
# Overview

Ensure that we are not mistaking commands that are issued by LPC before a run is started as commands
that should map to anticipated commands from the analysis

# Review requests

- [ ] Start a protocol after running the Labware Position Check. The commands list should maintain all of the expected commands

# Risk assessment
 low

### CODE REVIEW

File: app/src/organisms/RunDetails/CommandList.tsx [hunk 1]
```diff
@@ -41,68 +42,87 @@ export function CommandList(): JSX.Element | null {
...
+      firstPostPlayRunCommandIndex >= 0
+        ? runDataCommands.slice(firstPostPlayRunCommandIndex)
+        : []
+
+    console.log('PP ', postPlayRunCommands[0])
+    // const postPlayRunCommands = runDataCommands.filter(command => (
+    //   'createdAt' in command &&
+    //   command?.createdAt != null &&
+    //   new Date(command?.createdAt) < new Date(firstPlayTimestamp)
+    // ))
```
Comment (shlokamin) [inline app/src/organisms/RunDetails/CommandList.tsx]: leftover comments + console logs 🤪

File: app/src/organisms/RunDetails/CommandList.tsx [hunk 2]
```diff
@@ -41,68 +42,87 @@ export function CommandList(): JSX.Element | null {
...
       true
     )
 
     currentCommandList = isProtocolDeterministic
-      ? [...postSetupRunCommandSummaries, ...remainingAnticipatedCommands]
-      : [...postSetupRunCommandSummaries]
+      ? [...postPlayRunCommands, ...remainingAnticipatedCommands]
+      : [...postPlayRunCommands]
   }
+  console.log(currentCommandList[0], postSetupAnticipatedCommands[0])
```
Comment (shlokamin) [inline app/src/organisms/RunDetails/CommandList.tsx]: console log

**Review by shlokamin — APPROVED** [review]
LGTM, just a few comments + console logs to clean up. I don't have a robot so someone else should actually verify this works

**Review by jerader — APPROVED** [review]
tested on my robot, command list and everything is looking great 🚀

**Pull request merged**